private_sharing/forms.py

```diff
@@ -393,3 +424,37 @@ class DeleteDataFileForm(forms.Form):
     file_basename = forms.CharField(required=False, label="File basename")
 
     all_files = forms.BooleanField(required=False, label="All files")
+
+
+class SelectDatatypesForm(forms.Form):
```

Comment:
I think this form and associated view are more cleanly done with a ModelForm and UpdateView.